media/css/legal/impressum.less

```diff
@@ -0,0 +1,47 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../sandstone/lib.less";
+
+#tabzilla:before {
+    background-color: @textColorSecondary;
+}
+
+#main-feature {
+    > * {
```

Comment:
This is a bit broad and, seeing that the only element in the main-feature container is a `h1`. In fact, I reckon you can just get rid of the `main-feature` container and move the `h1` into the main-content section.